src/TpetraLinearSystem.C

```diff
@@ -1153,7 +1153,12 @@ void verify_row_lengths(const LinSys::Graph& graph,
          <<" entries: ";
       GlobalOrdinal rowGID = graph.getRowMap()->getGlobalElement(i);
       std::vector<GlobalOrdinal> vIndices(graph.getNumEntriesInGlobalRow(rowGID));
-      Teuchos::ArrayView<GlobalOrdinal> colIndices(vIndices);
+
+      Tpetra::CrsGraph<>::nonconst_global_inds_host_view_type colIndices("col_indices", vIndices.size());
+      for (size_t col_idx = 0; col_idx < vIndices.size(); ++col_idx) 
+        colIndices(col_idx) = vIndices[col_idx];
```

Comment:
I think this loop is unnecessary since vIndices wasn't initialized/filled yet, so no need to copy its contents.